src/plugin.js

```diff
@@ -222,7 +247,16 @@ const onPlayerReady = (player) => {
     // https://github.com/videojs/video.js/pull/4780
     // videojs.log('eme', 'Received an \'msneedkey\' event');
     setupSessions(player);
-    handleMsNeedKeyEvent(event, getOptions(player), player.eme.sessions, player.tech_);
+    try {
+      handleMsNeedKeyEvent(event, getOptions(player), player.eme.sessions, player.tech_);
+    } catch (error) {
+      emeError(error);
+    }
+  });
+  player.tech_.on('mskeyerror', emeError);
```

Comment:
Might be worth a quick comment to account for the different behavior saying that Promises are not supported in IE11.

Reply:
Added a comment a few lines up from here ^^^ is that what you meant? The behaviour (from a user perspective) should be the same in all browsers -- `error()` is called when there's an error at any point between start and ready to play.

Reply:
Yup, perfect 👍